.github/workflows/catalog-api-deploy.yml

```
@@ -0,0 +1,50 @@
+name: Deploy catalog-api
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types:
+    - deploy
+  workflow_run:
+    workflows: ["catalog-api"]
+    branches: [dev]
+    types: [completed]
+
+env:
+  chart: catalog-api
+  namespace: eshop
+  app-name: eshop
+  chart-root: deploy/k8s/helm
+
+jobs:
+  DeployToK8s:
```

```suggestion
  deploy-to-k8s:
```

I'd expect this to be named something like the suggestion above, instead of Hungarian notation.